packages/react/src/page-alert/docs/overview.mdx

```diff
@@ -113,3 +113,39 @@ Use the `onClose` prop to make the alert dismissable. Ensure Page alerts that ar
 	</Text>
 </PageAlert>
 ```
+
+## Focusing the alert
```

Comment:
I think you need to have an example of using `focusOnMount` and a manualy one like this

Reply:
- [x] example with manual + auto focusing